### PR TITLE
feat(accordion): creates accordion with pending issues

### PR DESCRIPTION
**Pull request by pedrohmorais**
## Description
https://jirasoftware.catho.com.br/browse/QTM-128

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] IE 11
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

File: components/Accordion/mock.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  margin-right: 8px;
+`;
+
+export default {
+  items: [
+    {
+      title: 'Institucional',
+      content: (
+        <div>
+          Blog <br />
```
Comment (renatogalvones) [inline components/Accordion/mock.jsx]: Adding some `List` examples would be nice.

File: components/Accordion/Accordion.jsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+  ${({ as }) => (as === 'button' ? `cursor: pointer;` : null)}
+`;
+
+const AccordionContent = styled.div`
+  height: ${({ opened }) => (opened ? 'auto' : '0')};
+  transform: scaleY(${({ opened }) => (opened ? '1' : '0')});
+  transform-origin: top;
+  overflow: hidden;
+  transition: height, transform 0.4s ease;
```
[the remark this answers is not in the document]
Comment (pedrohmorais) [reply components/Accordion/Accordion.jsx]: Done

File: components/Accordion/mock.jsx (hunk 1 above)
Comment (pedrohmorais) [reply components/Accordion/mock.jsx]: Done

**Comment by rapahaeru** [comment]
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [x] Layout review
  - [x] Edge
  - [x] IE 11
  - [x] Firefox
  - [x] Chrome
  - [ ] Safari
  - [ ] Mobile

**Pull request merged**